test/language/comments/hashbang-eval.js

```diff
@@ -0,0 +1,10 @@
+/*---
+esid: pending
+description: >
+    Hashbang comments should be available in Script evaluator contexts.
+info: |
+    HashbangComment::
+      #! SingleLineCommentChars[opt]
+---*/
+
+eval('#!\n');
```

Comment:
assert this completes to undefined?

Extra: maybe add another test file to evaluate custom completions?

Reply:
can you define "custom completions"?

Reply:
```js
assert.sameValue(eval('#!\n1`'), 1)
assert.sameValue(eval('#!2`'), undefined)
```


custom is a bad word, but any completion value that doesn't come from the hashbang comment